apps/web/src/components/nav/Sidebar.tsx

```diff
@@ -0,0 +1,37 @@
+import { useLocation } from 'react-router-dom';
+import { css } from '@novu/novui/css';
+import { ROUTES } from '../../constants/routes';
+import { LocalNavMenu } from '../../studio/components/LocalNavMenu';
+import { RootNavMenu } from './RootNavMenu';
+import { SettingsNavMenu } from './SettingsNavMenu';
+
+const className = css({
+  position: 'sticky',
+  top: 0,
+  zIndex: 'auto',
+  backgroundColor: 'transparent',
+  borderRight: 'none',
+  width: '272px',
+  height: '100%',
+  p: '100',
+  bg: 'surface.panel',
+  overflowY: 'auto',
+});
+
+export const Sidebar = () => {
+  const { pathname } = useLocation();
+
+  let Variant = RootNavMenu;
+
+  if (pathname.startsWith(ROUTES.SETTINGS)) {
+    Variant = SettingsNavMenu;
+  } else if (pathname.startsWith(ROUTES.STUDIO)) {
+    Variant = LocalNavMenu;
+  }
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** I'm not sure that this revision is an improvement over the original and will just add verbosity through a chain of `if / else if`‏

Reply:
I hear you. I considered adding a local Route component to handle the complicity but it felt too much for now. Settings will change very soon and I don't foresee more nesting in our sidebar but it's definitely something we need to refactor if sidebar becomes more complicated.